egg/zoo/compo_vs_generalization/callbacks.py

```diff
@@ -0,0 +1,47 @@
+import egg.core as core
+from egg.core.interaction import Interaction
+from egg.core.util import reset_optimizer_state
+from egg.zoo.compo_vs_generalization.curriculum_games import CurriculumGameWrapper
+
+import torch
+import numpy as np
+
+
+class CurriculumManager(core.Callback):
+    def __init__(self, 
+            game: CurriculumGameWrapper, 
+            optimizer: torch.optim.Optimizer,
+            acc_threshold=None):
+
+        self.game = game
+        self.optimizer = optimizer
+        self.acc_threshold = acc_threshold
+        self.acc_ors = []
+
+    def on_epoch_end(self, loss: float, logs: Interaction, epoch: int):
+        acc_or = logs.aux["acc_or"].mean().item()
+        update_curriculum = False
+        if self.acc_threshold is None:
```

Comment:
The fact that acc_threshold == None corresponds to a specific behaviour should be explained somewhere in a docstring